Microsoft.Identity.Web/Client/TokenAcquisition.cs

```diff
@@ -271,7 +198,7 @@ public async Task RemoveAccount(RedirectContext context)
         {
             ClaimsPrincipal user = context.HttpContext.User;
             IConfidentialClientApplication app = GetOrBuildConfidentialClientApplication(context.HttpContext, user);
-            IAccount account = await app.GetAccountAsync(context.HttpContext.User.GetMsalAccountId());
+            IAccount account = await app.GetAccountAsync(context.HttpContext.Request.Headers["Authorization"]);
```

Comment:
wow?

Reply:
?

---
In reply to: [318914112](https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/pull/93#discussion_r318914112) [](ancestors = 318914112)